prybar_assets/nodejs/module-context-hook.js

```diff
@@ -9,6 +9,28 @@ const nodeRepl = require('repl');
 const kReplitPrybarInit = Symbol.for('replit.prybar.init');
 const kReplitPrybarEvalCode = Symbol.for('replit.prybar.eval.code');
 const kReplitPrybarEvalFile = Symbol.for('replit.prybar.eval.file');
+let moduleMode = null;
+
+const originalFindPath = Module._findPath;
+Module._findPath = function(request, paths, isMain) {
+  if (moduleMode === "cjs" && request === 'node-fetch') {
+    // Because node-fetch version >= 3 doesn't support commonjs, we'll switch to
+    // node-fetch-commonjs if it comes to that
+    const result = originalFindPath(request, paths, isMain);
+    const dir = path.dirname(result);
+    const packageJson = require(path.join(dir, "..", "package.json"));
+    const [major] = packageJson.version.split(".").map(Number);
+    if (major >= 3) {
+      // Switch to node-fetch-commonjs
+      console.log("\u001b[33mAuto-switching node-fetch to node-fetch-commonjs\u001b[0m");
+      const newResult = result.replace('node_modules/node-fetch/src', 'node_modules/node-fetch-commonjs');
```

Comment:
we should bundle `node-fetch-commonjs`.  UPM won't automatically install this package.

Reply:
+1, we should set the path to the asset-dir, https://github.com/replit/prybar/blob/a584d8e52bbff356140546fc6691f1d2c01a176e/languages/nodejs/repl.js#L3-L8